app/uk/gov/hmrc/lisaapi/models/GetBulkPaymentResponse.scala

```diff
@@ -28,17 +28,18 @@ case object GetBulkPaymentErrorResponse extends GetBulkPaymentResponse
 case object GetBulkPaymentServiceUnavailableResponse extends GetBulkPaymentResponse
 
 trait BulkPayment
-
+case class finacialTransactions(paymentReferences: List[String])
```

Comment:
Is this needed? Don't see it being used.

Reply:
nope not needed